### PR TITLE
CRIS import ETL: check if crash report directory exists 

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues

- https://github.com/cityofaustin/atd-data-tech/issues/18952

## Testing

I have copied two of the problematic extracts in to the `./dev/cris_extracts/inbox` directory in S3. 

To test, start your local VZ stack and run the CRIS import. Please do not use the `--s3-archive` command so that others can test the s3-download.

```shell
# start the CRIS import container
$ docker compose run cris_import
# from inside the container shell 👇
root@docker-desktop:/app# ./cris_import.py --csv --pdf --s3-download --s3-upload
```

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [x] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-etl/cris_import/cris_import.py [hunk 1]
```diff
@@ -66,10 +67,22 @@ def main(cli_args):
...
+            """
+            crash_reports_dir = os.path.join(extract_dir, "crashReports")
+
+            if os.path.exists(crash_reports_dir):
+                records_processed["pdfs"] = process_pdfs(
+                    extract_dir, cli_args.s3_upload, cli_args.workers
+                )
+            else:
+                logger.warning(
+                    f"Warning: No 'crashReports' directory found — no PDFs to process"
```
Comment (mddilley) [inline atd-etl/cris_import/cris_import.py:83]: Yep, i saw this warning as well as the one about mismatch between crash count and pdf count with the test extracts.
```
WARNING: Warning: No 'crashReports' directory found — no PDFs to process
WARNING: Warning: there was a mismatch between the # of CSV crashes processed (2493) vs the CR3 PDFs processed (0).
```

**Review by mddilley — APPROVED** [review]
Tested the import, and I saw the new warning when running against the extract w/o pdfs. 🚢 🚀

**Review by frankhereford — APPROVED** [review]
For every twist from CRIS, you have coded up just the answer. This looks great and runs great -- thanks for the helpful testing instructions. 

# 🚢🚢🚢

**Comment by johnclary** [comment]
thank you, both! i may ship this as patch—we'll see 🍿 🙏

**Pull request merged**